cmd/hosts.go

```diff
@@ -55,7 +378,7 @@ func deploy(cmd *cobra.Command, args []string) error {
 
 	seq, err := store.TrustedClients()
 	if err != nil {
-		return err
+		fmt.Fprintf(os.Stderr, "Error: %v\n", err)
```

Comment:
Should return err here? The err is bubbled up to root.go Execute() where is will be caught and printed (again). However, by returning the execution is stopped here instead of continuing in case of error.